codespell_lib/data/dictionary.txt

```diff
@@ -18202,7 +18231,13 @@ fewg->few, fugue,
 fewsha->fuchsia
 fezent->pheasant
 fhurter->further
+fial->fail
```

Comment:
Perhaps:
```
fial->fail, dial, vial, final,
```

Reply:
Done. I also added `fially->finally` and `fialog->dialog`+friends